services/fastapi/routers/createsvg.py

```diff
@@ -47,7 +47,7 @@ async def execute_createsvg(request: Request, response: Response, body: CodeRequ
     # generate unique request id
     requestid = f'{datetime.timestamp(datetime.now())}.{str(random.randint(1, 100000))}'
     # get new name for a new script
-    new_output_name = f'req-{requestid}.svg'
+    new_output_name = f'req-{requestid}.html'
```

Comment:
is this a default behaviour or an option?

Reply:
Right now, this route is used for generating a d3 graph (the output viz), and for the tooltip algorithm, the output thing should be in HTML format, so I have changed the things from.svg to.html.